File: app/src/main/java/com/alex/materialdiary/sys/net/models/ClassicBody.java
package com.alex.materialdiary.sys.net.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ClassicBody {

    @SerializedName("apikey")
    @Expose
    private String apikey;
    @SerializedName("guid")
    @Expose
    private String guid;
    @SerializedName("date")
    @Expose
    private String date;
    @SerializedName("from")
    @Expose
    private String from;
    @SerializedName("to")
    @Expose
    private String to;
    @SerializedName("pdakey")
    @Expose
    private String pdakey;

    public String getApikey() {
        return apikey;
    }

    public void setApikey(String apikey) {
        this.apikey = apikey;
    }

    public String getGuid() {
        return guid;
    }

    public void setGuid(String guid) {
        this.guid = guid;
    }

    public String getDate() {
        return date;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public String getFrom() {
        return from;
    }

    public void setFrom(String from) {
        this.from = from;
    }

    public void setTo(String to) {
        this.to = to;
    }

    public String getTo() {
        return to;
    }

    public void setPdakey(String pdakey) {
        this.pdakey = pdakey;
    }

    public String getPdakey() {
        return pdakey;
    }

}

File: app/src/main/java/com/alex/materialdiary/sys/net/models/ClassicBody.kt
package com.alex.materialdiary.sys.net.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ClassicBody {
    @SerializedName("apikey")
    @Expose
    var apikey: String? = null

    @SerializedName("guid")
    @Expose
    var guid: String? = null

    @SerializedName("date")
    @Expose
    var date: String? = null

    @SerializedName("from")
    @Expose
    var from: String? = null

    @SerializedName("to")
    @Expose
    var to: String? = null

    @SerializedName("pdakey")
    @Expose
    var pdakey: String? = null

    @SerializedName("bias")
    @Expose
    var bias: Int? = null

    @SerializedName("areaname")
    @Expose
    var areaname: String? = null

    @SerializedName("blocksize")
    @Expose
    var blocksize: Int? = null
}